rmc/utils/mpc.py

```diff
@@ -305,9 +309,16 @@ def prepare_pop_path_ht(
     gnomad_ht = gnomad_ht.filter(gnomad_ht.freq[0].AF > af_threshold)
     gnomad_ht = gnomad_ht.annotate(pop_v_path=1)
 
-    logger.info("Joining ClinVar and gnomAD HTs...")
+    logger.info("Joining ClinVar and gnomAD HTs and filtering out overlaps...")
     ht = clinvar_ht.select("pop_v_path").union(gnomad_ht.select("pop_v_path"))
-    ht = ht.checkpoint(f"{TEMP_PATH}/joint_clinvar_gnomad.ht", overwrite=True)
+    # Remove variants that are in both the benign and pathogenic set
+    counts = ht.group_by(*ht.key).aggregate(n=hl.agg.count())
```

Comment:
```suggestion
    counts = ht.group_by(*ht.key).aggregate(n=hl.agg.count())
    counts = counts.checkpoint(f"{TEMP_PATH_WITH_DEL}/clinvar_gnomad_counts.ht", overwrite=True)
    overlap = counts.filter(counts.n > 1)
    logger.info("%i ClinVar P/LP variants are common in gnomAD", overlap.count()) 
```
this is mostly just because I'm curious, but it might be helpful to have this information logged for future reference

Reply:
I can't comment on the imports, but if you accept the above, we'd need to add an import for `TEMP_PATH_WITH_DEL`